build/commands/lib/config.js

```diff
@@ -284,6 +284,20 @@ Config.prototype.getBrandingPathProduct = function () {
   return this.isOfficialBuild() ? "brave" : "brave-development"
 }
 
+Config.prototype.getBraveLogoIconName = function () {
+  let iconName = "brave-icon-debug-color.svg"
```

Comment:
I'd say `debug` -> `dev`

Reply:
That file comes from leo module so can't use another one here.

Reply:
Filed https://github.com/brave/leo/issues/393

Reply:
this appears to break the thing that was fixed here https://github.com/brave/brave-core/pull/16033#discussion_r1027054377 and now we're copying files based on the current channel again. Why does upstream appear to have only one dark logo, but we have one for each channel?